frontend/src/data/parseReportData.test.ts

```diff
@@ -258,29 +266,93 @@ describe("parseReportData", () => {
         [
           1,
           {
-            description: case1.description,
-            schema: case1.schema,
-            tests: case1.tests,
+            description: testCases.case1.description,
+            schema: testCases.case1.schema,
+            tests: testCases.case1.tests,
           },
         ],
         [
           2,
           {
-            description: case2.description,
-            schema: case2.schema,
-            tests: case2.tests,
+            description: testCases.case2.description,
+            schema: testCases.case2.schema,
+            tests: testCases.case2.tests,
           },
         ],
         [
           3,
           {
-            description: case3.description,
-            schema: case3.schema,
-            tests: case3.tests,
+            description: testCases.case3.description,
+            schema: testCases.case3.schema,
+            tests: testCases.case3.tests,
           },
         ],
       ]),
       didFailFast: false,
     });
   });
+
+  test("parses report with dialect compliances", () => {
+    const cases = Object.values(testCases).map((each) => JSON.stringify(each));
+    const combinedData: Record<string, ReportData> = {};
+
+    for (const dialect of Dialect.known()) {
+      const lines = bowtie(
+        ["run", "-i", tag("envsonschema"), "-D", dialect.shortName],
+        cases.join("\n") + "\n",
+      );
+      const report = fromSerialized(lines);
+      combinedData[dialect.shortName] = report;
+    }
+
+    const parsedImplementationData = parseImplementationData(combinedData);
+    const metadata =
+      parsedImplementationData[tag("envsonschema")].implementation;
+
+    expect(parsedImplementationData).toStrictEqual({
+      [tag("envsonschema")]: {
+        implementation: {
+          name: "envsonschema",
+          language: "python",
+          homepage: metadata.homepage,
+          issues: metadata.issues,
+          source: metadata.source,
+          dialects: metadata.dialects,
+          links: metadata.links,
+        },
+        dialectCompliance: {
+          [Dialect.withName("draft2020-12").shortName]: {
```

Comment:
Dialect.withName("draft2020-12").shortName is just `draft2020-12` right? (And all the rest below)

Reply:
@Julian I’ll have to fix this test code once you merge PR #1058